autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -9,14 +9,63 @@
 import scipy.sparse
 
 import sklearn.utils
-from sklearn import preprocessing
+
 from sklearn.base import BaseEstimator
 from sklearn.compose import ColumnTransformer
 from sklearn.exceptions import NotFittedError
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.impute import SimpleImputer
+from sklearn.preprocessing import StandardScaler, OrdinalEncoder
 
 from autoPyTorch.data.base_feature_validator import BaseFeatureValidator, SUPPORTED_FEAT_TYPES
 
 
+def _create_column_transformer(preprocessors: typing.Dict, numerical_columns, categorical_columns):
```

Comment:
typing
I need the intention behind this function to review.

Reply:
yeah, this import typing was there from when fransisco coded it. I was too lazy to change these. I'll fix it in this file